src/kde_analysis/eval_m1m2chieff_poiss_weights.py

```diff
@@ -183,18 +187,15 @@ def get_rate_m_chieff2D(m1_query, m2_query, Rate):
     group = hdf[iter_name]
     if 'bootstrap_weights' in group:
         weighted = True
-        weights = group['bootstrap_weights'][:]
+        poisson_weights = group['bootstrap_weights'][:]
     samples = group['rwsamples'][:]
     alpha = group['alpha'][()]
     bwx = group['bwx'][()]
     bwy = group['bwy'][()]
     bwz = group['bwz'][()]
-
-    # Remove samples with zero bootstrap weight as they may have bad behaviour in
-    # adaptive KDE (due to extremely small pilot density at the sample location)
-    # and have no effect on the KDE, and to reduce compute cost
-    samples = samples[weights > 0., :]
-    weights = weights[weights > 0.]
+    # this include symmetric values 
```

Comment:
I think we will want to have the data saved consistently, so what is in the result file is without duplicating the values over m1<m2 and m1>m2 for all data sets.